### ISSUE TITLE
peity.js HTML

### ISSUE THREAD

**Issue by MayaGans** [JohnCoene/javascript-for-r-comments#9]
> As done with candidate libraries, let’s take a look at documentation of peity.js to see how it works.

Where did you find the piety.js script set up in an html page? How did you create it? The `index.html` in the `peity.js` library is way more complex, but I'd like to know how you set up this skeleton page to begin assessing what will be needed in your widget.

**Comment by JohnCoene** [comment]
I simplified it because the example in the official documentation is indeed rather convoluted :( Maybe I should rephrase this to be more clear?

> An example of using peity.js in a static HTML document....

**Comment by MayaGans** [comment]
I like that, or even explicitly stating that it's a paired down version from the repo - either way super helpful since that indicates looking at the `index.html` is one of the first steps in designing your widget :)

**Comment by JohnCoene** [comment]
Well, I think you sort of hint at something the book might lack though: how would one approach integrating a JavaScript library? Namely where to start (in the README?, from the website?) It's not easy to answer as you probably know it differs greatly from one to the other but perhaps a word on this should be in the book?
